api/libproject/devstart.rb

```diff
@@ -721,6 +700,19 @@ def migrate_database()
   end
 end
 
+def migrate_workbench_data()
```

Comment:
Do you think we'll need separate Ruby targets for these too?

Reply:
If it turns out that workbench has different data initialization needs than the CB... it might make sense to separate them. I think it's fine for now. Plus it will be easy to separate this into two tasks if need be.